pkgs/io_file/lib/src/file_system.dart

```diff
@@ -23,4 +61,12 @@ base class FileSystem {
   void rename(String oldPath, String newPath) {
     throw UnsupportedError('rename');
   }
+
+  /// Returns metadata for the given path.
```

Comment:
The method has a noun-phrased word, so it's a parameterized getter. Document it the same way.
Drop the "Returns". (Never start a DartDoc with the word "Returns". )

Maybe just `/// Metadata for the file or directory at [path].`

Reply:
Wrote as:

```dart
  /// Metadata for the file system object at [path].
```